packages/liveblocks-node/src/client.ts

```diff
@@ -88,8 +133,38 @@ export class Liveblocks {
   }
 
   /** @internal */
-  private async get(path: `/${string}`): Promise<Response> {
-    const url = urljoin(this._baseUrl, path);
+  private async put(
+    relativeURL: string,
+    json: Record<string, unknown>
```

Comment:
Consider using `JsonObject` (from `@liveblocks/core` here). The `unknown` type could contain values that aren't JSON-serializable, which would not be the user's intention.

Reply:
Good point. Would `Json` be more appropriate here, instead of `JsonObject`?

```ts
declare type Json = JsonScalar | JsonArray | JsonObject;
declare type JsonScalar = string | number | boolean | null;
declare type JsonArray = Json[];
declare type JsonObject = {
    [key: string]: Json | undefined;
};
```

Reply:
I suggested `JsonObject` here because you had a `Record` type here already. If you need to allow scalars and arrays here too, you can widen it to `Json`, sure. If not needed yet, I’d suggest starting with the strictest type that works first. We can always loosen it later if needed (but we cannot easily stricter it later).